src/strategies/jwt-strategy.js

```
@@ -2,15 +2,28 @@ const { Strategy: JwtStrategy, ExtractJwt } = require("passport-jwt");
 const prisma = require("../prisma");
 const passport = require("passport");
 
-const cookieExtractor = (req) => {
-	if (req?.cookies) {
-		return req.cookies.token; // Replace 'token' with the name of your cookie
+// Custom extractor untuk mengambil token dari cookie atau header
+const customExtractor = (req) => {
+	let token = null;
+	
+	// Cek di header Authorization
+	if (req && req.headers.authorization) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)